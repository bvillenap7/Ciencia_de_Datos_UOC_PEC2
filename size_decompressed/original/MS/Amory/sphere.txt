3 spheres spotted over Amory, MS while jets from Columbus AFB were training overhead.;2003-09-28 12:00:00;15 minutes;3 spheres spotted over Amory, MS while jets from Columbus AFB were training overhead. As usual I hear the roar of T-38's buzzing overhead as Columbus Airforce Base is just 40 miles south of Amory. On this day I happened to look up and I do sometimes to watch the aircraft play dogfight. Most days you cannot see anything because of the haze but every great once and a while you can... On this day I spotted 3 spheres together... they were in a cluster... I never saw them make any irradict moves nor did I ever see them separate.  They had a metallic look to their finish... since all of the jets were about 10k I can only assume these were somewhere in the same realm. I could not gestimate their size since I had nothing near by to reference them to.;33.9828;-88.4683
Duration from first sphere appearing to last disappearing approx. 7 minutes.;2013-11-28 17:55:00;7 minutes;Duration from first sphere appearing to last disappearing approx. 7 minutes On November 28 at approx.. 5:48 pm in Amory, Mississippi we {myself & my wife} were preparing to make our regular trip back home where we have been taking care of our mother since late March 2011, who is a 94 year old stroke victim and lives with us in Tennessee.  We take her home as much as we can {it seems to help her condition}   While I was taking out the garbage during a clean up process I noticed a very bright large white star in the west south-west sky, it was very beautiful.  I stared at it for a moment and then looked further to the west and gazed at a tiny star twinkling and then further west north-west an orange star began to appear it reminded me of the moon that every now and again turns yellow orange {you know}.   Peering through the trees out at the horizon the orange star was moving – “ oh well jet airliner   As it drifted higher like the ember from a fire another orange star began to appear, I was certain that the Columbus Air Force Base was conducting drills.  As the two orange lights rose slowly from the horizon-I was waiting for a familiar aerial sound-wop, wop, wop, but there was no sound and then another orange light appeared in the same area of orientation and began to rise into sky. As I stood there I certainly did not want to make a complete fool out of myself while another orange light began to rise.  My hopes were very anxious and high that a sonic boom or a popping sound would become proof that it was Columbus doing their thing. Only seconds had passed from the first light  to the fourth.   I turned and opened the kitchen door and called out to my wife and said honey come here and look at this and she I immediately came to where I was standing and said “WOW “its beautiful! What is it?”  Then another orange sphere begin to rise from the same location as all the others came from.    As we peered through the trees my wife said I bet we can see them better in the front yard so we went to the front yard and the view was a little better as the orange lights were rising higher and were heading due north and then another orange sphere began to appear from the same area of orientation and then another and then another it was like a roman candle in a slow m! otion rotation pattern.   Somewhere higher than the others in a drift pattern you would expect the embers of a fire swirling in the wind.  My wife said “there is no sound maybe its those balloons they put candles in and release on a wish!   OK I was starting to buy into the idea and suddenly all the lights became in alignment shaped like a v or as my wife put it – like a row of geese but the alignment was moving due north and the point of the shape was pointing south.   They began quickly to disappear and my wife ran inside the house to get a camera to possibly photograph them.  There were only 3 light left – 1 alone and 2 together.  These images were sent to you and now there is a written recording of our memory. ((NUFORC Note:  Witness elects to remain totally anonymous provides no contact information.  PD));33.9828;-88.4683
