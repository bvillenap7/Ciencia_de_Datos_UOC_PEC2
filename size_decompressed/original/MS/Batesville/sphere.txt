Shiny Silver Sphere seen for about 10 minutes in rural North Mississippi circa 1975;1975-10-15 16:00:00;10 min;Shiny Silver Sphere seen for about 10 minutes in rural North Mississippi circa 1975 It was around October 1975 in rural North Mississippi, four miles north of Batesville. I was 16 years old at the time and was walking along the edge of a soybean field with my trusty .22 rifle squirrel hunting.  I heard a very loud high pitched buzzing sound (like a cicada in the trees--which it may have been) but then everything was silent--even the birds. I looked up over my shoulder and saw a silver sphere just hovering about 300 feet above the ground and 1/4 mile away. It just stayed there. I thought, hey look, a weather balloon. but there wasn't anything hanging from the bottom. Not a weather balloon. No gondola, so it also wasn't a hot air balloon either. It was just there hovering silently.  Here's where I acted like a typical teenager. The .22 had a scope on it, so I looked at the sphere through the scope--looked like a shiny silver ball. I decided, what the heck I squeezed off a shot thinking if it was a balloon, I would be able to retrieve it. It hovered for a few seconds more then slowly drifted up and out of sight. There were cumulus clouds in the sky that day and the sphere was going along the edge of them rising in a slightly erratic fashion. But I was able to keep it in sight while it rose due to the contrast of the white clouds.  The thing that still makes me wonder to this day is how readily I accepted what I saw and how I felt no alarm ((NUFORC Note:  Witness indicates that date of incident is approximate.  PD));34.3115;-89.9443000000001
