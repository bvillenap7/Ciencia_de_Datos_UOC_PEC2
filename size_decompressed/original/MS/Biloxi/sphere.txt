Sighting of UFO by Fernwood Junior High School band students. Biloxi, Mississippi between 1976 and 1979.;1977-10-01 19:00:00;2-3 minutes;Sighting of UFO by Fernwood Junior High School band students.  Biloxi, Mississippi, between 1976 and 1979. I have never reported this story and have only shared it with my closest family and friends.  Those who know me well enough to know I do not lie nor do I embellish stories. Between August, 1976 and May, 1979, I attended Fernwood Junior High School, Pass Road, Biloxi, Mississippi, 39531, for my 7th, 8th and 9th grade years.  So It has been nearly 40 years since my UFO experience.  That being said, though I am a little vague on the exact date or time  the circumstances are still as clear in my mind as if they had happened yesterday. One evening I was among a group of kids who had just finished band practice.  There were probably 7-9 of us waiting for our parents to pick us up.  If I remember correctly it was for concert band and it had to have been at least 8:00pm or 8:30pm, maybe a little later than that.  After practice was finished we were all huddled at the back of the school where it looked over the  football field.  On the other side of the field was an area of woods and past that lie the beaches.  It was completely dark outside and I don’t remember being cold so I believe it was in the late summer or early fall sometime between September and December.  The sky was completely clear and illuminated by only the stars and the moon with a faint glow from some street lights on the front side of the school (north of our position) and from the businesses which lined Beach Blvd, approximately a mile away on the other side of the woods, to the south.  Biloxi was east of or location and Gulfport was to our west.  Also lying east of our location was Keesler Air Force Base.  My father was stationed there. We were all gathered and just talking.  When someone in the group noticed a bright light that appeared at the top of the trees.  Since it was on the other side of the field I estimate it was approximately 200 feet +/- away from or location and based on the its position above the field I estimate that it had an approximately diameter of 40-60 feet.  It was a bright white sphere that was completely silent and slowly glided just above the tree line towards the direction of Gulfport until it was out of our line of sight.  It only took about 30 seconds for it to disappear past the tree tops. After a moment of silent disbelief a buzz of talking incurred with questions about the source of the light but what happened next really sent the conversation into a frenzy.  As I said the sphere was completely silent but approximately 20 seconds after it disappeared a group of helicopters came into view at the top of the trees to our east.  They were not in a straight line but in some kind of formation spread out from the tree line towards the beach.  We heard them loud and clear before we even saw them (which squashed the point that the Sphere was too far away for us to hear) and they followed the same exact path across the top of the trees, east to west, hurriedly scanning below with search lights, along their path, as they passed the school.   About that time we heard our parents blow their car horns and we went running up to the front of the school. We loaded into the car (I was riding with a friend) and excitedly began to describe our experience. We were headed East on Pa! ss Road towards home. I lived in Thrower Park, which was base housing.  No sooner had we started driving down Pass Road when 4-5 police cars, with lights and sirens blaring, sped past us in the direction of Gulfport. This only fueled our speculations.  The next day I looked in the newspapers for any word of the events of the previous evening but there was nothing.  Of course we spoke of it at school and I kept watch in the papers for several weeks but eventually gave up on finding out what happened that night. Over the years I have thought of our experience often and wondered about the truth but in my mind I have no doubt about what we saw.  During my father’s 20 year career in the Air Force he worked on mission specific equipment for aircraft and I was in and around aircraft frequently when I would visit the hangers where he worked.  Some of the mission equipment he worked on was for the Hurricane Hunters and I was very aware of what a weather balloon was and what it looked like.  So much so that as a child I wanted to grow up to be a weather person. I was captivated by weather and the Hurricane Hunters.  We had also lived in Albuquerque, New Mexico, and been to the hot air balloon festivals on several occasions, during my childhood. I wish I could say I remember the date and the names of the other kids who were present that night but other than my recollection of the experience and a few nicknames I am at a loss.  How I wish we had the internet then.  I may not have been able to find any information back then but I have never forgotten and would love to know if there is anyone else out there, who went to Fernwood Junior High during that period, and who has not forgotten either. ((NUFORC Note:  Witness indicates that the date of the event is approximate.  PD));30.4555575757576;-88.9453060606062
Unidentified Sphere observed by group of teens west of Keesler AFB, Biloxi, MS.;1978-11-01 20:00:00;2 minutes;Unidentified Sphere observed by group of teens west of Keesler AFB, Biloxi, MS. I have never reported this story and have only shared it with my closest family and friends.  Those who know me well enough to know I do not lie nor do I embellish stories. Between August, 1976 and May, 1979, I attended Fernwood Junior High School, Pass Road, Biloxi, Mississippi 39531 for my 7th, 8th and 9th grade years.  So It has been nearly 40 years since my UFO experience.  That being said, though I am a little vague on the exact date or time  the circumstances are still as clear in my mind as if they had happened yesterday. One evening I was among a group of kids who had just finished band practice.  There were probably 7-9 of us waiting for our parents to pick us up.  If I remember correctly it was for concert band and it had to have been at least 8:00pm or 8:30pm, maybe a little later than that.  After practice was finished we were all huddled at the back of the school where it looked over the  football field.  On the other side of the field was an area of woods and past that lie the beaches.  It was completely dark outside and I don’t remember being cold so I believe it was in the late summer or early fall sometime between September and December.  The sky was completely clear and illuminated by only the stars and the moon, with a faint glow from some street lights on the front side of the school (north of our position) and from the businesses which lined Beach Blvd, approximately a mile away on the other side of the woods, to the south.  Biloxi was east of or location and Gulfport was to our wes! t.  Also lying east of our location was Keesler Air Force Base.  My father was stationed there.  We were all gathered and just talking.  When someone in the group noticed a bright light that appeared at the top of the trees.  Since it was on the other side of the field I estimate it was approximately 200 feet +/- away from or location and based on the its position above the field I estimate that it had an approximately diameter of 40-60 feet.  It was a bright white sphere that was completely silent and slowly glided just above the tree line towards the direction of Gulfport until it was out of our line of sight.  It only took about 30 seconds for it to disappear past the tree tops. After a moment of silent disbelief, a buzz of talking occurred, with questions about the source of the light but what happened next really sent the conversation into a frenzy.  As I said, the sphere was completely silent but approximately 20 seconds after it disappeared a group of helicopters came into view at the top of the trees to our east.  They were not in a straight line but in some kind of formation spread out from the tree line towards the beach.  We heard them loud and clear before we even saw them (which squashed the point that the Sphere was too far away for us to hear) and they followed the same exact path across the top of the trees, east to west, hurriedly scanning below with search lights, along their path, as they passed the school.  About that time we heard our parents blow their car horns and we went running up to the front of the school.  We loaded into the car (I was riding with a friend) and excitedly began to describe our experience.  We were headed East on Pa! ss Road towards home.  I lived in Thrower Park, which was base housing.  No sooner had we started driving down Pass Road when 4-5 police cars, with lights and sirens blaring, sped past us in the direction of Gulfport.  This only fueled our speculations.  The next day I looked in the newspapers for any word of the events of the previous evening, but there was nothing.  Of course we spoke of it at school and I kept watch in the papers for several weeks, but eventually gave up on finding out what happened that night. Over the years I have thought of our experience often and wondered about the truth but in my mind I have no doubt about what we saw.  During my father’s 20 year career in the Air Force he worked on mission specific equipment for aircraft and I was in and around aircraft frequently when I would visit the hangers where he worked.  Some of the mission equipment he worked on was for the Hurricane Hunters and I was very aware of what a weather balloon was and what it looked like.  So much so that as a child I wanted to grow up to be a weather person.  I was captivated by weather and the Hurricane Hunters.  We had also lived in Albuquerque, New Mexico, and been to the hot air balloon festivals on several occasions, during my childhood. I wish I could say I remember the date and the names of the other kids who were present that night, but other than my recollection of the experience and a few nicknames I am at a loss.  How I wish we had the internet then.  I may not have been able to find any information back then but I have never forgotten and would love to know if there is anyone else out there, who went to Fernwood Junior High during that period, and who has not forgotten either. ((NUFORC Note:  Witness indicates that the date of the event is approximate.  PD));30.4555575757576;-88.9453060606062
Biloxi Ms Sighting;2005-02-25 17:30:00;12 minutes;Biloxi Ms Sighting At about 5:30P.M. I looked up in the sky and saw what looked to be a sphere/slight cigar shape metallic object very high up almost in space.  I alerted two of my friends that were standing in my yard with me and they saw it also.  I ran inside and got my telescope and while i was gone my two friends said they saw it speed up and then come almost to a dead stop.I couldnt focus my telescope in on it because of the bright glare of the sun that it was reflecting but all three of us watched as it slowly went from north to south eventually disappearing below the tree line.;30.4555575757576;-88.9453060606062
