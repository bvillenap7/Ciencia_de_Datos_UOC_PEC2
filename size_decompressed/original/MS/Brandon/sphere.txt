We saw a Black Sphere moving South at a good rate of speed over Rankin County just South of the Ross Barnett Reservoir.  No Sound, No W;2015-01-21 16:45:00;60 second;We saw a Black Sphere moving South at a good rate of speed over Rankin County, just South of the Ross Barnett Reservoir.  No sound, no wind.  Airport is to the West.  I walked outside and saw it, called my wife and she also saw it just before it went out of sight because of the trees.   I first thought it was a balloon, then a noticed that there was NO wind and it was moving.  (DID NOT HAVE TIME TO TAKE A PHOTO) observers are (Sales Manager) and (Teacher);32.3515982142858;-89.9588785714287
Bright mini sized moon appeared near dusk in the northern sky for about 5 minutes and then zoomed out of sight;2021-05-03 19:30:00;5 minutes;Bright mini sized moon appeared near dusk in the northern sky for about 5 minutes and then zoomed out of sight Greetings, I’m an automation specialist and have been working in the field of electronic technology for over 50 years. I am 70 years old. I have seen a few odd things in the sky in my life but this was the most unusual and unexplained.  I’m reporting this recent sighting because I have seen nothing about it on any news outlet. I was so shocked by what I saw that I thought it would be global news.  This is my account of what I saw in the sky on May 3, 2021 in Brandon, Ms. I was outside in the early evening working on a new front porch addition. I noticed that it was 7:30 pm and looked up to the sky as I was sitting down to see if I could see bats flying overhead. I had noticed bats coming out to eat insects the evening before at that same time and wanted to see if they were out again and they were. As I was looking to the north, I saw a very bright object. I thought maybe it was Venus because it was so bright and big but it was in the wrong place and this object had an odd flare coming out of one side of it almost like a comet trail but very short. It sort of looked like what you would see if you threw paint ball on a flat surface and it splattered a bit in one direction– round with a few rays and a bit smudged.  When I noticed the flare, I ran inside to get my binoculars and lost a minute or two looking. All I could fine was my oversize pair so I grab them and went back outside. The object was still there and had grown larger but the flare was gone. I tried to focus on it. My binoculars are very large and hard to handle without a mount but I was able to focus on it and I was shocked into disbelief. I had no idea what I was looking at.  It looked like miniature moon about 3 or 4 time the size of Venus as observed from earth and just as bright.  It was bright silver with shaded areas and absolutely round.  There was not another light in the sky – no stars, no planets and the sky was still filled with light from the sun as it was setting. There was a storm rolling in, a tornado watch and the sky was a bit hazy. I doubt that there were many people in my area outside looking up at the time.  I put the binoculars down and picked up my phone to bring up my astronomy app to see if I could identify what it was that I was seeing. As I was fiddling with my phone to bring up the app ( I have to take my case off because it has a magnet on it and that destroys the accuracy of the compass and it doesn’t work for positioning)  the object started dimming and shrinking in size as if it was now heading away from my perspective. When I saw it dim, I knew the app wouldn’t recognize this strange object because it was nothing normal about it.  I decided to try to get a picture of it before it was gone and I did snap a few pictures before it disappeared but the contrast between the object and the sky was low for a camera because it was still pretty much daylight. I accidentally took 2 one second videos and they came out better than the pictures. I put the phone down and watched the object as it narrowed down to a point of light and vanished. When the object was heading away it remained perfectly round as shrunk in size and brightness.  In the two short videos you can see the effect of the atmosphere as it distorts the shape of the object. I am an amateur astronomer and I am very familiar with atmospheric distortion and the distortion I see in the videos tells me the object was above our atmosphere. I wish I could have gotten some photos of it when I first saw it because it was much bigger plus it had a flare at first but I was stunned and was kind of wondering if this was doomsday.  My first thought when I saw the object in my binoculars was that something huge was about to hit earth unannounced. It looked like a rouge moon headed directly for us. When it started moving away, I felt relief as it shrunk in size. I knew that it must be traveling at an extreme speed the way it disappeared. The whole event lasted about 5 minutes. The object pretty much stayed in the same location in the sky and did not jump or dart around. I didn’t see it come into view – it was just there when I looked up and it only moved a little in position when it took off. There was no sound, no flashing light and the only colors I saw was sliver and gray. Since it was hazy during the observation, I have to question the detail that I saw on the object. I now assume that maybe I was seeing cloud vapor super imposed on the object and that vapor gave it the darker areas in the disk I viewed that resembled the terrain of the moon. Now my question is – Did anybody else see this object?   My guess was that it was an asteroid making a very close fly-by but the news and the astronomy groups are saying nothing about it.  It’s seems impossible that no one else noticed it. It’s a complete mystery to me.;32.3515982142858;-89.9588785714287
