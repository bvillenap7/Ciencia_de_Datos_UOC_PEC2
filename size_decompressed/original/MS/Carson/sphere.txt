same as 4/3/09 	Sumrall, Ms;2009-05-01 05:10:00;10min;same as 4/3/09 	Sumrall, Ms Almost the same as http://www.nuforc.org/webreports/069/S69520.html but the object was a red sphere with around 5 or 6 smaller red objects around it. It moved slowly east for around 3-5min and then back west picking up speed for 1-2min then back east,disappearing over the treeline.;31.485;-89.8299
