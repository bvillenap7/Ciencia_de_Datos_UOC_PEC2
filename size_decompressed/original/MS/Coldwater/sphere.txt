Orange orbs in ms;2013-09-28 20:55:00;2 minutes;Orange orbs in ms Was headed home going down 306 and spotted 15 to 20 orange orbs coming over the tree tops headed south west.  I was on my bike and the person in front of me was driving really slow, they had to have seen them too.  I had to pull over and try to get a video of them.  By the time I got my phone out most if them had already went over the trees. I caught four of them with my iPhone.  I started looking into the orange orb phenomena and come to find out these orbs are being seen every where.  There are great videos of them on YouTube people have caught.;34.6937;-89.9912
