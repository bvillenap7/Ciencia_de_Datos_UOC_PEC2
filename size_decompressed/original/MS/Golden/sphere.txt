Multiple small, white and blue orbs gather in a large tree nearby after i flashing my light toward where i seen the first orb.;2020-05-13 23:00:00;20 minutes;Multiple small, white and blue orbs gather in a large tree nearby after i flashing my light toward where i seen the first orb. I noticed small white flashes above my yard as I grilled. Shortly after, I saw a blue, three strobe orb fly a few feet above my house. I was completely shocked because I knew instantly that it was nothing I had seen before. As I stood there flipping my burgers, thinking about what i had just seen I had an idea.  I had my flashlight on my phone on, so I flashed my light 3 times up in the sky like whatever I had just seen fly overhead. Nothing happened the first time but when i done it again the thing came back around. This time it parked and hovered in a very tall tree not far from my house. It wasn't flashing anymore but a constant dim blue light slowly weaving through the branches of this very large tree.  As I continued flashing my light in flashes of three, I began to see 2 other small white lights in the same tree as the blue orb. They made no sound at all. Couldn't of been a drone because it was in this large oaktree, moving around in the dense leaves. They looked sort of like fireflies but there are no fireflies out yet.  It was at about this time my wife came out and I filled her in. I tried repeatedly to get the lights to flash again but nothing. But sure enough as I was about to give up, all three lights began to move and light up! So my wife seen these lights and I knew this was something that was really there, maybe 100 yards from my house hanging in this very large tree. Doing what? I have no clue. But after she saw the lights she became frightened and she made me quit communicating with my light and go inside.  This was the first time I have experienced anything like this, ever.;34.4775;-88.1822
